src/Generator.cpp

```diff
@@ -1312,12 +1318,14 @@ Pipeline GeneratorBase::get_pipeline() {
     return pipeline;
 }
 
-std::string GeneratorBase::auto_schedule_outputs(const MachineParams &arch_params) {
-    return get_pipeline().auto_schedule(get_target(), arch_params);
+void GeneratorBase::auto_schedule_outputs(const MachineParams &arch_params) {
+    user_assert(auto_schedule_result.empty()) << "auto_schedule_outputs was called multiple times for " << generator_registered_name;
```

Comment:
Why is it not okay to call auto_schedule_outputs multiple times?

Reply:
Hmm nvm, I forgot that the generated schedule is applied directly on the funcs which make calling it again invalid since auto-scheduler doesn't handle partially scheduled pipeline